diff --git a/model/faceDetector/s3fd/__init__.py b/model/faceDetector/s3fd/__init__.py
index 943292a..a029f3d 100644
--- a/model/faceDetector/s3fd/__init__.py
+++ b/model/faceDetector/s3fd/__init__.py
@@ -6,7 +6,7 @@ from torchvision import transforms
 from .nets import S3FDNet
 from .box_utils import nms_
 
-PATH_WEIGHT = 'model/faceDetector/s3fd/sfd_face.pth'
+PATH_WEIGHT = './thirdparty/humanvbench_models/Light-ASD/model/faceDetector/s3fd/sfd_face.pth'
 if os.path.isfile(PATH_WEIGHT) == False:
     Link = "1KafnHz7ccT-3IyddBsL5yi2xGtxAKypt"
     cmd = "gdown --id %s -O %s"%(Link, PATH_WEIGHT)

diff --git a/model/faceDetector/s3fd/box_utils.py b/model/faceDetector/s3fd/box_utils.py
index 0779bcd..1bf4be2 100644
--- a/model/faceDetector/s3fd/box_utils.py
+++ b/model/faceDetector/s3fd/box_utils.py
@@ -35,7 +35,7 @@ def nms_(dets, thresh):
         inds = np.where(ovr <= thresh)[0]
         order = order[inds + 1]
 
-    return np.array(keep).astype(np.int)
+    return np.array(keep).astype(int)
 
 
 def decode(loc, priors, variances):
